cider/tests/utils/QueryArrowDataGenerator.h

```diff
@@ -308,6 +316,25 @@ class QueryArrowDataGenerator {
     std::generate_n(str.begin(), length, randchar);
     return str;
   }
+
+  static std::string gen_date_format_string(size_t length) {
```

Comment:
same above, also dedup with this part?
https://github.com/intel/BDTK/blob/7f18c39f4cddaee713e0c4017081a3f70b0cddb0/cider/tests/utils/QueryDataGenerator.h#L268-L286

Reply:
`QueryDataGenerator` should be removed/deprecated soon. And also we don't need to rely on` CiderByteArray` to construct data.